### DISCUSSION TITLE
More generic templates

### DISCUSSION THREAD

**Discussion by JSXRED** [eusonlito/Password-Manager#63] [Ideas]
I would like to see 2 more "generic" templates in the applications section. Only "password" and only "username "+"password" .

For example, if I want to store the password for a ZIP file, the password for a PFX certificate or similar, I have hardly any possibility to do so. 

Also the pure storage of e.g. system wide service accounts is difficult. Currently, one can help out with the WIFI template, but the somewhat misappropriated. 

We noticed in everyday life that we miss these two generic templates, because not all passwords have a URL, IP or similar.

**Comment by eusonlito** [comment]
Moved to issue :)

https://github.com/eusonlito/Password-Manager/issues/64
